docs/guides/upgrading.rst

```diff
@@ -401,6 +401,8 @@ Notable changes in plugins:
  * Support for ``.elgg-icon-hover`` has been dropped
  * Admin theme now reuses icon classes from ``elements/icons.css``
 
+Also note, CSS views served via ``/cache`` URLs are pre-processed using `CSS Crush <http://the-echoplex.net/csscrush/>`_. If you make references to CSS variables or other elements, the definition must be located within the same view output. E.g. A variable defined in ``elgg.css`` cannot be referenced in a separate CSS file like ``colorbox.css``.
+
```

Comment:
in a future PR we will introduce a generic view with variables that gets injected into the csscrush api, this way you can use global (core) variables in all files..